### ISSUE TITLE
Mass import all example machines into docs

### ISSUE THREAD

**Issue by StoneCypher** [StoneCypher/fsl#1102]
Blocked by #1054

**Comment by StoneCypher** [comment]
Superseded — folded into the FSL example/proof corpus (#817). Closing per the approved triage ledger (`notes/superpowers/eras/dispositions/fsl-triage.md`).
